include/api.php

```diff
@@ -114,12 +114,12 @@ function pll_default_language( $field = 'slug' ) {
  *
  * @param int                 $post_id Post ID.
  * @param PLL_Language|string $lang    Optional language (object or slug), defaults to the current language.
- * @return int|false The translation post ID if exists, otherwise the passed ID. False if the passed object has no language or if the language doesn't exist.
+ * @return int|false The translation post ID if exists, 0 if not translated or if the post has no language. False if the language doesn't exist.
  *
  * @phpstan-return int<0, max>|false
  */
-function pll_get_post( $post_id, $lang = '' ) {
-	$lang = $lang ? $lang : pll_current_language();
+function pll_get_post( $post_id, $lang = null ) {
```

Comment:
Should `null` be added in the `@param $lang` in doc ?

Reply:
The doc states `defaults to the current language` which seems enough to me.
Default to `null` is not self explanatory IMHO.